docs/content/docs/dev/table/sql-gateway/rest.md

```diff
@@ -0,0 +1,119 @@
+---
+title: REST Endpoint
+weight: 2
+type: docs
+aliases:
+- /dev/table/sql-gateway/rest.html
```

Comment:
Please add a "HiveServer2 Endpoint" page, but soft link to `/dev/table/hiveCompatibility/hiveserver2.html`.